ui/tests/integration/components/auth/form/oidc-jwt-test.js

```diff
@@ -74,28 +74,6 @@ const authUrlRequestTests = (test) => {
     assert.true(firstRequest, 'it makes FIRST request to auth_url with default path');
     assert.true(secondRequest, 'it makes SECOND request to auth_url with custom path');
   });
-
-  test('it re-requests auth_url when input changes: namespace', async function (assert) {
-    // request assert should be hit twice, once on initial render and again on namespace change
-    assert.expect(2);
-    this.server.post(`/auth/${this.authType}/oidc/auth_url`, () => {
-      assert.true(true, 'it makes request to auth_url');
-      return { data: { auth_url: '123-example.com' } };
-    });
-    await render(hbs`
-      <Auth::Form::OidcJwt 
-        @authType={{this.authType}} 
-        @cluster={{this.cluster}}
-        @onError={{this.onError}}
-        @onSuccess={{this.onSuccess}}
-      >
-        <:namespace>
-          <label for="namespace">Namespace</label>
-          <input data-test-input="namespace" id="namespace" name="namespace" type="text" /> 
-        </:namespace>
-      </Auth::Form::OidcJwt>`);
-    await fillIn(GENERAL.inputByAttr('namespace'), 'mynamespace');
-  });
 };
```

Comment:
This is a query param now correct so changing it triggers a route refresh?

Reply:
`namespace` has always been a query param, but yes updating it re-instantiates the component so it's redundant to also re-fetch `auth_url` from within the component. It also caused unexpected race conditions because sometimes this request would fire with the outdated namespace. Happy to go into more detail if you'd like 😄

Reply:
Gotcha that makes sense to me thanks!